src/Libraries/CoreNodes/DateTime.cs

```diff
@@ -358,6 +361,11 @@ public static System.TimeSpan FromString(string str)
         ///     Extracts the individual components of a TimeSpan.
         /// </summary>
         /// <param name="timeSpan">A TimeSpan.</param>
+        /// <returns name="days">Amount of days on TimeSpan</returns>
+        /// <returns name="hours">Amount of hours on TimeSpan</returns>
+        /// <returns name="minutes">Amount of minutes on TimeSpan</returns>
+        /// <returns name="seconds">Amount of seconds on TimeSpan</returns>
+        /// <returns name="milliseconds">Amount of milliseconds on TimeSpan</returns>
```

Comment:
Do we need the `on TimeSpan` on all of these?

Reply:
Addressed in 983b5dc